src/chrome/content/rules/Wdr.de.xml

```diff
@@ -1,22 +1,79 @@
 <!--
+	No valid Cert for:
 
-	Problematic domains:
+		- podcast.wdr.de
+		- print.wdr.de
+		- reportage.wdr.de
+		- www.einslivekrone.de
+		- www.kiraka.wdr.de
+		- www.kirche.wdr.de
+		- www.lindenstrasse.de
+		- www.wdrmaus.de
+		- ^wdr.de
 
-		- www1 ¹
+	certificatechain broken on:
+		- webmail.wdr.de
+		- freundeskreis.einslive.de
 
-	¹ Redirect to HTTP
+	Non-resolvable host:
+		- static.freundeskreis.einslive.de
```

Comment:
It is part of the SAN part of the certificate for www.wdr.de so for the sake of completeness i noted that